src/pivot-table/components/cells/ColumnAdjuster.tsx

```diff
@@ -21,14 +22,18 @@ interface AdjusterProps {
  * While dragging this components follows the pointer, and on mouse up all column widths are updated.
  */
 const ColumnAdjuster = ({ cellInfo, columnWidth, dataModel, isLastColumn }: AdjusterProps) => {
+  const { isActive } = useSelectionsContext();
   const [internalWidth, setInternalWidth] = useState(columnWidth);
   const tempWidth = useRef({ initWidth: 0, columnWidth: 0, initX: 0 });
   const positionAdjustment = isLastColumn ? CELL_PADDING : CELL_PADDING + GRID_BORDER;
+  const shouldRender = !isActive && cellInfo.canBeResized;
 
   useOnPropsChange(() => {
-    setInternalWidth(columnWidth);
+    if (shouldRender) setInternalWidth(columnWidth);
   }, [columnWidth]);
 
+  if (!shouldRender) return undefined;
```

Comment:
Return `null` when we don't want to render a componet. Not sure if React threats it differently by that is the value they use in the docs.

Reply:
yeah I was expecting a type error if it was incorrect, will change to null